legend-engine-xt-serviceStore-grammar/src/main/antlr4/org/finos/legend/engine/language/pure/grammar/from/antlr4/connection/ServiceStoreConnectionLexerGrammar.g4

```diff
@@ -2,7 +2,26 @@ lexer grammar ServiceStoreConnectionLexerGrammar;
 
 import CoreLexerGrammar;
 
+@lexer::members{
+  static int lastTokenType=0;
+public void emit(Token token) {
+  super.emit(token);
+  this.lastTokenType = token.getType();
+}
+}
+
 // -------------------------------------- KEYWORD --------------------------------------
 
 STORE:                                  'store';
-BASE_URL:                               'baseUrl';
\ No newline at end of file
+BASE_URL:                               'baseUrl';
+AUTH_SPECS:                             'auth';
```

Comment:
no shorthands?
Also do we really need this complexity ?

Reply:
changed it to authentication